web-dev/src/components/Profile/index.js

```diff
@@ -27,11 +31,9 @@ const ProfileScreen = () => {
                         {/* Log Out Button */}
                         {/* Need to log user out of their profile via authentication / React Context */}
                         <span className="ms-3">
-                            <Link to="/login">
-                                <button className="wd-landing-sign-out-button fs-6">
+                                <button type="button" className="wd-landing-sign-out-button fs-6" onClick={() => void signOut()}>
```

Comment:
With AuthenticationLock, just trigger the signOut() from auth-context navigation will be done for you

Reply:
So when we use AuthenticationLock we need to also use signOut() somewhere to kick the user out?